src/enkf/enkf_obs_sensitivity.f90

```diff
@@ -48,7 +49,7 @@ module enkf_obs_sensitivity
 use convinfo, only: convinfo_read,init_convinfo
 use ozinfo, only: ozinfo_read,init_oz
 use radinfo, only: radinfo_read,jpch_rad,nusis,nuchan,npred
-use gridinfo, only: latsgrd,lonsgrd,nlevs_pres,npts
+!use gridinfo_efsoi, only: latsgrd,lonsgrd,nlevs_pres,npts,id_u,id_v
```

Comment:
@AndrewEichmann-NOAA Is this commented line needed?  Is the original gridinfo line needed instead?

Reply:
@CatherineThomas-NOAA I was incorrect in what I said at the meeting about this change - it wasn't part of my cleanup of copied modules. Blame says Dave made this change in the EXP-efsoi_fv3 branch three years ago. None of the variables that gridinfo references are currently used in this module, so it should probably be removed anyway. I will delete the commented line in the PR branch.